cranelift-simplejit/tests/basic.rs

```diff
@@ -195,3 +195,13 @@ fn libcall_function() {
 
     module.finalize_definitions();
 }
+
+#[test]
+#[should_panic]
```

Comment:
Please add an expected panic message: https://doc.rust-lang.org/reference/attributes/testing.html?highlight=should_panic#the-should_panic-attribute

Reply:
Will do :)